src/services/types.ts

```diff
@@ -631,7 +631,9 @@ export interface LanguageService {
     applyCodeActionCommand(fileName: string, action: CodeActionCommand | CodeActionCommand[]): Promise<ApplyCodeActionCommandResult | ApplyCodeActionCommandResult[]>;
 
     getApplicableRefactors(fileName: string, positionOrRange: number | TextRange, preferences: UserPreferences | undefined, triggerReason?: RefactorTriggerReason, kind?: string): ApplicableRefactorInfo[];
+    getMoveToRefactoringFileSuggestions(fileName: string, positionOrRange: number | TextRange, preferences: UserPreferences | undefined, triggerReason?: RefactorTriggerReason, kind?: string): { newFilename: string | undefined, files: string[] | undefined };

```

Comment:
Nit: capitalization of “n” in `fileName` vs. `newFilename`. We seem to capitalize more often than not.